packages/queued-request-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.1.0]
+
+### Removed
+
+- `createQueuedRequestMiddleware` no longer takes a `useRequestQueue` parameter. All requests are now queued if `shouldEnqueueRequest(req)` returns true. ([#4941](https://github.com/MetaMask/core/pull/4941))
```

Comment:
Removing this option will cause a type error if a client is still attempting to pass it, so should we mark it as a breaking change and release it under 8.0.0?

Reply:
Ah yes I suppose thats true. I'm used to doing this in JS in extension so never get these errors 😅